src/components/list/List.tsx

```diff
@@ -1,12 +1,36 @@
+import { SetStateAction, useState } from "react";
 import { getCollection } from "../../firebase/config";
 import { useFetchCollection } from "../../hooks/useFetchCollection";
 import { PointType } from "../../types";
 import { ListItem } from "./ListItem";
+import { query, where } from "firebase/firestore";
+
+// type Props = {
+//   typeOptions: SelectOption[];
+// };
 
 export const List = () => {
+  // const {
+  //   register,
+  //   formState: { errors },
+  // } = useForm<PointNewFormSchemaType>({
+  //   resolver: zodResolver(PointNewFormSchema),
+  // });
+
   const { data, isLoading, error } = useFetchCollection<PointType>(
-    getCollection("point")
+    query(getCollection("point"), where("type", "in", ["louka", "les", "park"]))
   );
+  const [filter, setFilter] = useState("all");
+  const filteredData = data?.filter((point) => {
```

Comment:
tohle vubec nepotrebujes ne? data dostanes z BE vfiltrovane. 

Zkusil bych `  const { data, isLoading, error } = useFetchCollection<PointType>(
    query(getCollection("point"), where("type", "in", filter))` a nebo ve fci poresis jeste is timall statem treba tahle `getFilters = () => filter === 'all' ? query(getCollection("point")) : query(getCollection("point"), where("type", "in", filter))`